ulib/experimental/Steel.Effect.Common.fsti

```diff
@@ -1608,8 +1639,8 @@ let init_resolve_tac () : Tac unit =
   // To debug, it is best to look at the goals at this stage. Uncomment the next line
   // dump "initial goals";
 
-  // We now solve all postconditions of pure returns to avoid restricting the uvars
-  solve_all_returns (goals ());
+  // // We now solve all postconditions of pure returns to avoid restricting the uvars
+  // solve_all_returns (goals ());
```

Comment:
Delete this too? Also curious how come we don't need it anymore?

Reply:
This was previously needed when introducing a Steel.return. return is post-parametric, but if it has effect Steel (and not SteelF), an additional frame would have been introduced leading to two frame uvars in a constraint. The solution was this preprocessing phase, where all return post-slprops were solved before entering the core tactic, by "capturing" the full context, and solving all possible associated frames to emp.

With the current changes, return always has the framed bit set to true, which corresponds to only allowing SteelF returns in the previous formalism. Hence this preprocessing is not needed anymore